package.json

```diff
@@ -73,7 +73,7 @@
     "strip-comments": "^2.0.1",
     "terser": "^5.9.0",
     "unistore": "^3.4.1",
-    "web-vitals": "^2.1.3",
+    "web-vitals": "^2.1.4",
```

Comment:
Generally this is fine, but did you just change the number in the JSON or did you `npm i web-vitals@2.1.4`

I would have thought the package-lock.json would have updated as well, which I think we need (i feel like things can get wonky with out automated builds without it).

Reply:
Ahh, right, I forgot to `npm install` after changing the version. Updated!